config/config_schema.json

```diff
@@ -341,6 +341,78 @@
                     ]
                 }
             }
+        },
+        "oidcAuthProvider": {
+            "title": "OpenID Connect AuthProvider",
+            "description": "OpenID Connect authentication configuration",
+            "type": "object",
+            "additionalProperties": false,
+            "required": [
+                "idpUrl",
+                "uniqueField",
+                "clientId",
+                "scope"
+            ],
+            "properties": {
+                "idpUrl": {
+                    "description": "Base URL for identity provider endpoint",
+                    "type": "string",
+                    "format": "uri",
+                    "pattern": "^https?://",
+                    "examples": [
+                        "https://keycloak.ilifu.ac.za/auth/realms/ilifu"
```

Comment:
I don't think we should make the example point to our actual endpoint, because users might use autocomplete to fill fields in and then spam our endpoint